src/backup/utils.ts

```diff
@@ -95,16 +96,17 @@ export function formatBackupPhraseOnEdit(phrase: string) {
   return phrase.replace(/\s+/gm, ' ')
 }
 
-export function formatBackupPhraseOnSubmit(phrase: string) {
-  return normalizeMnemonic(phrase)
+export function isValidBackupPhrase(phrase: string) {
+  const allowedPhraseLengths = [12, 24]
+  const phraseLength = countMnemonicWords(formatBackupPhraseOnEdit(phrase))
+  return !!phrase && allowedPhraseLengths.includes(phraseLength)
 }
 
-function isValidMnemonic(phrase: string, length: number) {
-  return !!phrase && countMnemonicWords(formatBackupPhraseOnEdit(phrase)) === length
-}
+export function getDerivationPath(phrase: string) {
+  const wordCount = countMnemonicWords(phrase)
+  const derivationPath = wordCount === 24 ? CELO_DERIVATION_PATH_BASE : ETHEREUM_DERIVATION_PATH
 
-export function isValidBackupPhrase(phrase: string) {
```

Comment:
also combined isValidMnemonic and isValidBackupPhrase, seemed to do the same thing